src/autoscaler/metricsgateway/helpers/ws_helper_test.go

```diff
@@ -108,10 +108,9 @@ var _ = Describe("WsHelper", func() {
 			err = wsHelper.Ping()
 			Expect(err).ShouldNot(HaveOccurred())
 			Eventually(pingPongChan, 5*time.Second, 1*time.Second).Should(Receive(Equal(1)))
-
 		})
 		It("close the websocket connection", func() {
-			wsHelper.CloseConn()
+			err = wsHelper.CloseConn()
 			Expect(err).NotTo(HaveOccurred())
```

Comment:
this was checking the error that was set in errr before the action ... maybe we should do 2 checks ?

Reply:
the previous error is checked in the `BeforeEach` func